ligo/liquidity_mining.mligo

```diff
@@ -0,0 +1,665 @@
+// SPDX-FileCopyrightText: 2021 Arthur Breitman
+// SPDX-License-Identifier: LicenseRef-MIT-Arthur-Breitman
+
+// TODO: left a license note to the stacker-v3 repository?
```

Comment:
So, the `v3-staker` is licensed under GPL3, which requires modification of the code to be carrying the same licence:
>   if you distribute copies of such a program, whether
gratis or for a fee, you must pass on to the recipients the same
freedoms that you received.

see also [here](https://choosealicense.com/licenses/gpl-3.0/).

So, it could be argued if this is the same source, since you re-implemented completely, but I guess it is given that this is entirely based on the solidity source AFAIU.

You already give enough information about the source material and where to find it, the only thing left to do is to change this to:
```suggestion
// SPDX-License-Identifier: LicenseRef-GPL3-Arthur-Breitman
```
(and add the corresponding file to `/LICENSES`)